app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person["name"], person["age"]) for person in people]
+
+    for person in people:
+        if person.get("wife"):
+            wife = Person.people[person.get("wife")]
+            Person.people[person.get("name")].wife = wife
```

Comment:
.get does not need inside if. Because we know that wife key exist and we know that name is exist also
Fix the same in husband

Reply:
Should I follow the rule "use get method to avoid KeyError" in this case?

Reply:
Should I follow the rule "use get method to avoid KeyError" in this case?